python/tests/integration/connectors/memory/test_qdrant_memory_store.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import numpy as np
+import pytest
+
+from semantic_kernel.connectors.memory.qdrant import QdrantMemoryStore
+from semantic_kernel.memory.memory_record import MemoryRecord
+
+try:
+    import qdrant_client  # noqa: F401
+
+    qdrant_client_installed = True
+except ImportError:
+    qdrant_client_installed = False
+
+pytestmark = pytest.mark.skipif(
+    not qdrant_client_installed, reason="qdrant-client is not installed"
+)
+
+
+@pytest.fixture
+def qdrant_memory_record():
+    return MemoryRecord(
+        id="test_id1",
+        text="sample text1",
+        is_reference=False,
+        embedding=np.array([0.5, 0.5]),
+        description="description",
+        external_source_name="external source",
+        timestamp="timestamp",
+    )
+
+
+def test_qdrant_constructor():
+    qdrant_mem_store = QdrantMemoryStore
```

Comment:
I don't think this runs without providing a host addr

Reply:
Qdrant has added ability to run for python without host addr, I'll need to add that to this test. Thanks for the catch.

Reply:
Addressed in commit: https://github.com/microsoft/semantic-kernel/pull/919/commits/b1b6de71171c6bfca8448c4238831794ccb87ff6